exchanges/kline/kline_types.go

```diff
@@ -40,6 +40,7 @@ const (
 	ThreeMonth           = 90 * OneDay
 	SixMonth             = 2 * ThreeMonth
 	OneYear              = 365 * OneDay
+	FiveDay              = 5 * OneDay
```

Comment:
There is a merge issue with OneMonth please revert it back to `30 * OneDay`.  Please add fiveday to supported list.